tests/test_utils.py

```diff
@@ -1040,3 +1042,46 @@ def working_directory(directory: PathStr):
         yield directory
     finally:
         os.chdir(owd)
+
+
+class MockSaInspector:
+    def get_columns(self) -> list[dict[str, Any]]:  # type: ignore[empty-body]
+        ...
+
+    def get_schema_names(self) -> list[str]:  # type: ignore[empty-body]
+        ...
+
+    def has_table(self, table_name: str, schema: str) -> bool:  # type: ignore[empty-body]
+        ...
```

Comment:
Mock `SA` classes are moved to `test_utils` and tests have been updated as a result.

Reply:
Could we move this instead to a file with a name that is more descriptive? Something like `sqlalchemy_test_doubles.py` or something like that? Will help with finding / organizing things later.

Reply:
Great idea. I'll do this now